src/main/kotlin/no/nb/mlt/wls/domain/ports/outbound/ItemRepository.kt

```diff
@@ -13,6 +13,15 @@ interface ItemRepository {
     fun createItem(item: Item): Mono<Item>
 
     suspend fun doesAllItemsExist(ids: List<ItemId>): Boolean
+
```

Comment:
I know it's unrelated, but can we fix the grammar of the above metod to "doAllItemsExist" or "doesEveryItemExist"?